modules/nlu/src/migrations/v12_7_1-1583507028-prune_and_compress_models.ts

```diff
@@ -1,6 +1,95 @@
 import * as sdk from 'botpress/sdk'
+import fse, { WriteStream } from 'fs-extra'
```

Comment:
old migration file...

I simply pasted the previous model repo to make sure the migration would still work as it used to